src/AddIns/Uno.UI.MSAL/Uno.UI.MSAL.net6.csproj

```diff
@@ -0,0 +1,85 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFrameworks>net6.0-android;net6.0-ios</TargetFrameworks>
+		<TargetFrameworksCI>net6.0-android;net6.0-ios</TargetFrameworksCI>
+
+		<TargetFrameworks>$(TargetFrameworks);net6.0-maccatalyst</TargetFrameworks>
+		<TargetFrameworksCI>$(TargetFrameworks);net6.0-maccatalyst</TargetFrameworksCI>
+	</PropertyGroup>
```

Comment:
Instead of duplicating the file, maybe there's a way to put common stuff in a shared file?

Reply:
There aren't that much that are exactly duplicated unfortunately... but yes, we could share some of it for many projects. I'd rather not change existing projects for now, the PR already large enough.